node/src/service.rs

```diff
@@ -251,6 +253,7 @@ where
 			relay_chain_interface: relay_chain_interface.clone(),
 			import_queue: params.import_queue,
 			net_config,
+			sybil_resistance_level: cumulus_client_service::CollatorSybilResistance::Unresistant,
```

Comment:
Could we be resistent?

Reply:
@aliXsed 
From the code [here](https://github.com/paritytech/polkadot-sdk/blob/e53ebd8cd48a42f053bb8c30bbe84c90065ce3c8/cumulus/client/service/src/lib.rs#L387)



```
pub enum CollatorSybilResistance {
	/// There is a collator-selection protocol which provides sybil-resistance,
	/// such as Aura. Sybil-resistant collator-selection protocols are able to
	/// operate more efficiently.
	Resistant,
	/// There is no collator-selection protocol providing sybil-resistance.
	/// In situations such as "free-for-all" collators, the network is unresistant
	/// and needs to attach more metadata to block announcements, relying on relay-chain
	/// validators to avoid handling unbounded numbers of blocks.
	Unresistant,
}
```

I read this as we should mark if we are resistant or not. So I think we should pick `Unresistant` here